### PR TITLE
odds api ajax call working

### PR DESCRIPTION
**Pull request by sleepytomatoes**
everything that is commented out is a rough plan for how I expect the ajax call to go in our application.  I'm still figuring out the data in the response object for accuracy but so far I am able to get the data I want from the DOM tree.  Some of the Ids I've added just for placeholders since we need the html from Ethan to test.  Could also use your test html file to do this in future Jason.

### CODE REVIEW

**Review by codragon2020 — CHANGES_REQUESTED** [review]
I think there are missing lines of code in the index.html file.  My current git pull shows the Error Modal has been merged.  Looks like you may have forgotten to pull your latest changes before you updated the index file.  

The other file is fine to add as is.

File: index.html [hunk 1]
```diff
@@ -127,8 +127,9 @@ <h2 class="modal-title">Instructions:</h2>
 
     <!-- double check this is the right Bootstrap link for Javascript -->
     <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js" integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1" crossorigin="anonymous"></script>
-    <script src="app.js"></script>   
-
+    <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
+    <!-- <script src="app.js"></script>    -->
+    <script src="testodds.js"></script>
```
Comment (codragon2020) [inline index.html:133]: This should be appended to line 147.

**Comment by sleepytomatoes** [comment]
redoing with updated index.html
